interactive/api/static/examples/CHAPMAN/camp_data/config.json

```diff
@@ -1,7 +1,7 @@
 {
   "camp-files" : [
-    "camp_data/tolerance.json",
-    "camp_data/species.json",
-    "camp_data/reactions.json"
+    "species.json",
+	"reactions.json",
+    "tolerance.json"
```

Comment:
The order shouldn't matter. If it does, we need to correct whatever code thinks the order matters

Reply:
We talked about the order on Slack. There is python code that requires a certain order, and we have plans to fix that. For now, I have put the files in the expected order and this sequence will be forward-compatible.